docs/en/ingest-management/release-notes/release-notes-8.5.asciidoc

```diff
@@ -13,13 +13,36 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.5.1>>
 * <<release-notes-8.5.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.5.1 relnotes
+
+[[release-notes-8.5.1]]
+== {fleet} and {agent} 8.5.1
+
+Review important information about the {fleet} and {agent} 8.5.1 release.
+
+[discrete]
+[[bug-fixes-8.5.1]]
+=== Bug fixes
+
+{fleet}::
+* Make asset tags space aware {kibana-pull}144066[#144066]
+
+{fleet-server}::
+* No bug fixes for this release
+
+{agent}::
+* No bug fixes for this release
```

Comment:
There are a few changes in 8.5.1 for agent based on the changelog fragments, which look correct: https://github.com/elastic/elastic-agent/tree/8.5/changelog/fragments

Additionally, there is a beats bug fix I think we should highlight in the agent release notes: https://github.com/elastic/beats/pull/33169 

> - Beats will now attempt to recover if a lockfile has not been removed {pull}[33169]

The fix is in Beats but the bug primarily happens when Beats run as part of agent, and the problem solved here has been actively causing Beats run by agent to report as unhealthy for a while.